src/pages/Game.tsx

```diff
@@ -73,50 +70,34 @@ const playersInit = (currentGame: Game, pixiStage: PIXI.Container, currentUser:
 const GamePage = ({ ws, currentUser }: GameProps): JSX.Element => {
   const canvasRef = useRef<HTMLDivElement>(null);
 
-  /**
-   * Allows animate to redraw the canvas if there is a state change.
-   */
-  const requestRedraw = () => {
-    redrawPending = false;
-  };
-
   /**
    * Game reference is passed from GameLobby through history.push
    */
   const history = useHistory();
   const { state } = history.location;
   const { game } = state as { game: Game };
 
+  /**
+   * Callbacks required for adding interaction to PIXI assets.
+   */
+  const wsCallbacks: Record<string, (...args: unknown[]) => void> = {
+    [OutgoingAction.PLAY_TILE]: (tile: unknown) => {
+      ws?.sendMessage(OutgoingAction.PLAY_TILE, { game: game.gameId, tile });
```

Comment:
Up to you. Would it be more efficient if client knows which gameId to broadcast instead of getting it from user connectionId -> gameId

Reply:
Probably keep it consistent with your code and not pass in gameId

Reply:
my bad, I do need gameId for the PLAY_TILE lambda, so it would be:
```
ws?.sendMessage(OutgoingAction.PLAY_TILE, { gameId: game.gameId, tile });
```

Reply:
Will edit the flowchart